generated/ruby/google-cloud-ruby/google-cloud-error_reporting/google-cloud-error_reporting-v1beta1.gemspec

```diff
@@ -0,0 +1,28 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |s|
+  s.name          = 'google-cloud-error_reporting-v1beta1'
```

Comment:
Hey - sorry for the delay on this. We don't want the version number in the package name.